anvil/src/input_handler.rs

```diff
@@ -586,6 +587,40 @@ impl AnvilState<UdevData> {
         }
     }
 
+    fn on_pointer_move_absolute<B: InputBackend>(&mut self, dh: &DisplayHandle, evt: B::PointerMotionAbsoluteEvent) {
+        let serial = SCOUNTER.next_serial();
+
+        let max_x = self
+            .space
+            .outputs()
+            .fold(0, |acc, o| acc + self.space.output_geometry(o).unwrap().size.w);
+        let max_y = self
+            .space
+            .outputs()
+            .fold(0, |acc, o| acc + self.space.output_geometry(o).unwrap().size.h);
```

Comment:
This is not entirely correct. Outputs in smithay only extend to the right, so the maximum-y coordinate is the largest monitor, but not the sum of all.